src/styles/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: $calendar-width;
+  padding: $padding;
```

Comment:
Ensure that the SCSS variables `$gap`, `$calendar-width`, and `$padding` are defined elsewhere in your SCSS files. If they are not defined, this will cause a compilation error.